Justfile

```diff
@@ -186,6 +195,17 @@ format-frontend:
 pnpm-install:
   @for i in {1..3}; do mk frontend/**/node_modules : frontend/**/package.json -- "pnpm install --frozen-lockfile" && break || sleep 5; done
 
+# Install Rust dependencies based on Cargo.toml
+rust-toolchain-install:
+    @mk sqlc-gen-ftl/target : sqlc-gen-ftl/Cargo.toml -- \
+        "cd sqlc-gen-ftl && \
+        rustup-init -y --no-modify-path --default-toolchain stable && \
+        cargo install protoc-gen-prost && \
```

Comment:
We could also get rid of this by adding a `scripts/protoc-gen-prost` something like this:

```bash
#!/bin/bash
mk .hermit/rust/bin/protoc-gen-prost -- cargo install protoc-gen-prost     
exec .hermit/rust/bin/protoc-gen-prost "$@"
```

Then we wouldn't need the "rust-toolchain-install" step at all, everything would self-install.

Reply:
removed all the `rust-toolchain-install` stuff with the addition of `rust-toolchain.toml`

Reply:
ah sorry got your comment mixed up - awesome will add that